portfolio/src/main/webapp/pages/gallery.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Gallery</title>
+    <link rel="stylesheet" href="../style.css">
+  </head>
+  <body>
+    <ul class="topnav">
+      <li><a href="../index.html">Isabel Mitre</a></li>
+      <li class="right"><a href="about.html">ABOUT ME</a></li>
+      <li class="right"><a href="projects.html">PROJECTS</a></li>
+      <li class="right"><a href="gallery.html">GALLERY</a></li>
+      <li class="right"><a href="../index.html">HOME</a></li>
+    </ul>
+    <div class="content3">
+      <div id="explanantion">
+        <h1>Garden Gallery</h1>
+        <p> I love my plants. Unfortunately, I am a bad plant mother and frequently forget to 
+            water my plants. Fortunately, it is the wet season here in South Florida so they're
+            getting plenty of water regardless. They're also pretty resistant. The cotton plants 
+            are probably three years old and I have some citrus trees (seen in the butterfly and 
+            chrysalis pictures) that are probably twice that.
+        </p>  
+      </div> 
+      <div class="column">
+        <img class="gallery" src="../images/gallery/butterfly.jpg"/>
```

Comment:
Can you add alternatives for the images? It can be really simple, like `alt="Butterfly".` (This is important for accessibility reasons, there's more details in the style guide: https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback)

Reply:
done!